File: Swift-C++-IR-Comparison/test/stringsAndCharacters/Swift/SpecialCharacters.swift
@main
struct MyMain {

    static func main() {
        let wiseWords = "\"Imagination is more important than knowledge\" - Einstein"
        // "Imagination is more important than knowledge" - Einstein
        let dollarSign = "\u{24}"  // $,  Unicode scalar U+0024
        let blackHeart = "\u{2665}"  // ♥,  Unicode scalar U+2665
        let sparklingHeart = "\u{1F496}"  // 💖, Unicode scalar U+1F496

        let threeDoubleQuotationMarks = """
            Escaping the first quotation mark \"""
            Escaping all three quotation marks \"\"\"
            """
    }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-1/SpecialCharacters.cpp
 
#include<iostream>
using namespace std;
int main()
{
	string wiseWords = "\"Imagination is more important than knowledge\" - Einstein";
	string dollarSign = "\u0024"; // $; Unicode scalar U+0024
	string blackHeart = "\u2665" // ♥; Unicode scalar U+2665
	string sparklingHeart = "\u1F496"; // 💖; Unicode scalar U+1F496

	cout << wiseWords;
	cout << dollarSign;
	cout << blackHeart;
	cout << sparklingHeart;

	return 0;
}